### ISSUE TITLE
Incorrect error for bad argument of a DOM type

### ISSUE THREAD

**Issue by sandersn** [Microsoft/TypeScript#27049]
```ts
declare var ohno: new () => Node;
declare function ff(t: Node): void;
ff(ohno)
```

**Expected behavior:**
Error: "Argument of type 'new () => Node' is not assignable to parameter of type 'Node'

**Actual behavior:**
Type 'new () => Node' is not assignable to type 'Node'.

Only seems to repro when both the source and target types are in the DOM.

**Comment by sandersn** [comment]
Could be caused by #27016 or #27013?

**Comment by DanielRosenwasser** [comment]
Probably the latter - it'd be great if these baselines included related error spans.

**Comment by weswigham** [comment]
Yeah, definitely the latter. As is, an elaboration overrides the default error message, even if the elaboration occurs right at the top level. It's fixable, if we'd like to fix it.
